ansible/roles/stack-sunbird/templates/sunbird_nodebb.env

```diff
@@ -1,10 +1,15 @@
 # Initialization variable for mongo
 # Ref: https://community.nodebb.org/topic/4325/set-up-nodebb-via-environment-variables
-database=mongo
+database=redis
+# redis database number
+redis_database={{nodebb_redis_database|default('0')}}
+# declearing js heap memory size
+nodebb_heap_size={{nodebb_heap_size|default('2048')}}
 secret={{nodebb_hash_key|default('1d57ba64-86d4-43ff-bd10-f6e9e0782899')}}
 url=http://0.0.0.0:4567/discussions/
-mongo__host={{mongo_nodebb_host}}
-mongo__database=nodebb
+redis__host={{sunbird_lp_redis_host }}
```

Comment:
Please reach out to @rjshrjndrn and decide on which redis should we use. KP / DP?

Reply:
KP looks good to me.

Reply:
Added username and password for redis db